src/main/kotlin/wordle/view/OutputView.kt

```diff
@@ -0,0 +1,34 @@
+package wordle.view
+
+import wordle.domain.Answer
+import wordle.domain.Color
+
+object OutputView {
+    private val results: MutableList<List<Color>> = mutableListOf()
+
+    fun printIntroduction() {
+        println("WORDLE을 6번 만에 맞춰 보세요.")
+        println("시도의 결과는 타일의 색 변화로 나타납니다.")
+    }
```

Comment:
이건 애쉬에게 리뷰받았던 내용인데 여기서 인자를 받을 수 있으면 총 라운드 수가 바뀌어도 뷰를 유지할 수 있을 것 같아요!
```suggestion
    fun printIntroduction(round : Int) {
        println("WORDLE을 {$round}번 만에 맞춰 보세요.")
        println("시도의 결과는 타일의 색 변화로 나타납니다.")
    }
```

Reply:
오 좋네요! 변경했습니다~